### DISCUSSION TITLE
How to get access to copilot (issue with invite?)

### DISCUSSION THREAD

**Discussion by matthewjablack** [community/community#10185] [Copilot Conversations] [answered]
I signed up several weeks ago and have still not received an invite. Everyone on my team has gotten access already and I'm wondering if there's an issue with my invite.

**Comment by diimdeep** [comment]
Hi, I signed up 7 months ago, at the time of discussion on [Hacker News](https://news.ycombinator.com/item?id=27676266) 
I didn't receive invite. I remember that in questionnaire I chose that don't use VS Code much, maybe that is why in my case there still no invite. Maybe unlucky.

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see [#9342 (comment)](https://github.com/github/feedback/discussions/9342#discussioncomment-1902060)
